packages/sdk-js/test/json.test.ts

```diff
@@ -67,6 +68,8 @@ type Cases = {
     gt: [string, string, boolean][];
     eq: [string, string, boolean][];
   };
+  // name, context, experiment, result
+  urlRedirect: [string, Context, AutoExperiment, string][];
```

Comment:
Pass in experiments via the `Context` and make result an array of objects so we can check to make sure that tracking callbacks are only fired for one experiment rather than multiple if there is more than one experiment targeting the same url